source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,42 @@
+.phone {
+  $ratio: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(black, 0.8);
+  $phone-padding: 0.4em * $ratio;
+  $phone-screen-width: 8.5em * $ratio;
+  $phone-screen-height: 15em * $ratio;
+  $phone-background: #F8F8F8;
+
+  background: $phone-background;
+  border-radius: (1.2em * $ratio);
+  box-shadow: 0 0 1px transparentize(black, 0.4), inset 0 3px 0 white;
+  padding: $phone-padding;
+  width: $phone-screen-width + (2 * $phone-padding);
+
+  .phone-speaker {
+    background: #ABABAB;
+    border-radius: 1em;
+    box-shadow: 0 2px 0 white, inset 0 0 1px transparentize(black, 0.6);
+    height: (0.3em * $ratio);
+    margin: (1em * $ratio) auto;    
+    width: $phone-screen-width * 0.2;
+  }
+
+  .phone-screen {
+    background: url('https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/cosmin_capitanu_screen.jpg');
+    background-size: cover;
+    border-radius: 0.2em;
+    box-shadow: 0 3px 0 white, inset 0 0 1px transparentize(black, 0.6);
+    height: $phone-screen-height;
+    margin: auto;
+    width: auto;
+  }
+
+  .phone-button {
+    border-radius: 50%;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, height, margin, width